components/timeline/assets/Timeline.scss

```diff
@@ -0,0 +1,86 @@
+@import "@not-govuk/sass-base";
+@import "govuk-frontend/govuk/helpers/_colour.scss";
+
+.hods-timeline {
+  overflow: hidden;
+  position: relative;
+
+  p {
+    text-decoration: inherit;
+    font-size: inherit;
+  }
+
+  &__item {
+    @include govuk-responsive-padding(2, "bottom");
+    @include govuk-responsive-padding(5, "left");
+    /* padding size 6 looks better with smaller viewports. Can media queries
+    be used here? */
+    position: relative;
+
+    &:hover {
+      background-color: #E3E3E3;
+    }
+
+    &:before {
+      background-color: #1d70b8;
```

Comment:
Magic constant!

Can this be turned into a variable?

Reply:
Should this be inherited from the styles in the gov toolkit?

Reply:
If it exists there then yes.

Reply:
Robin has pointed out that we have this problem elsewhere.

If the value isn't in one of the colour palletes and isn't close enough to something in the pallette then this can be left as it is.

If the value is repeated throughout the file it would be _nice_ to turn it into a variable so that it can be controlled in one place.

Sorry for raising an issue that we already have in the rest of the code-base. It's partly due to me reviewing this in a rush.

I'm happy to resolve this thread if you like.

Reply:
As mentioned above, `#1d70b8` is the same colour as `govuk-colour("blue")`. Have updated it in the latest commit.